graph/cytoscape/cytoscape.go

```diff
@@ -22,9 +22,19 @@ import (
 	"github.com/kiali/kiali/graph/options"
 )
 
+// ProtocolTraffic.Responses is a map of maps. Each response code is broken down by responseFlags:percentageOfTraffic, e.g.:
+// "200" : {
+//    "-"     : 80.0,
+//    "DC"    : 10.0,
+//    "FI,FD" : 10.0
```

Comment:
Technically, this comment is incorrect - since the map is [string]string, the values here are also strings, not numbers... so it should be:
```
"-"     : "80.0",
"DC"    : "10.0",
"FI,FD" : "10.0"
```

Reply:
+1, I'll fix it.